provider/resource_keycloak_realm_keystore_java_kyestore_test.go

```diff
@@ -0,0 +1,274 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+	"regexp"
+	"strconv"
+	"testing"
+)
+
+func TestAccKeycloakRealmKeystoreJava_basic(t *testing.T) {
+	t.Parallel()
+
+	skipIfEnvSet(t, "CI") // temporary while I figure out how to put java keystore file to keycloak container in CI
```

Comment:
we could probably just mount the `misc` directory as a volume on the running container and reference it that way.  but I'm fine with leaving this as-is for now.

Reply:
Yes, I tried to to this(using the `misc` dir) but the problem that I need keystore file inside keycloak instance, not in runner instance. 

In the terraform manifest inside `keystore` field I paste path for keycloak itself.